test/Psy/Test/ConfigurationTest.php

```diff
@@ -238,4 +239,22 @@ public function testSetColorModeInvalid()
         );
         $config->setColorMode($colorMode);
     }
+
+    public function testSetCheckerInvalid()
+    {
+        $config = new Configuration();
+
+        $this->setExpectedException('TypeError');
```

Comment:
The rest of the project uses the annotation style for expected exceptions. I don't have a strong preference, but we should prolly keep it consistent :)